source/server/spongineer/commands.rst

```diff
@@ -200,6 +201,14 @@ There are also extra permissions managing the access to the server:
 * Treat the user as whitelisted: ``minecraft.login.bypass-whitelist``
 * Allow this user to bypass the server's player limit: ``minecraft.login.bypass-player-limit``
 
+.. note::
+
+    Sponge offers improved multi-world support, such as per world world borders. By default Sponge only changes the
+    world border (or other world options) of the world the player is currently in. The vanilla behavior of setting it
+    for all worlds can be restored using the global configuration and setting
+    ``sponge.commands.multi-world-patches.worldborder`` or the corresponding entry to ``false``. See 
```

Comment:
Because it also affects the `gamerule` and a few other commands as well. 
(Ref: https://github.com/SpongePowered/SpongeDocs/issues/505#issuecomment-411146517)

Should I use a plural here as well?  

>or the corresponding entries

Reply:
No, I was more confused by context. That bit might look better in brackets (to match the context above).